### PR TITLE
ACS-3275: Expose Rules V1 APIs

### PR DESCRIPTION
**Pull request by KikoUK**
https://alfresco.atlassian.net/browse/ACS-3275

### CODE REVIEW

File: src/main/java/org/alfresco/rest/model/RestRuleModelsCollection.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.alfresco.rest.model;
+
+import org.alfresco.rest.core.RestModels;
+
+public class RestRuleModelsCollection extends RestModels<RestRuleModel, RestRuleModelsCollection>
```
Comment (tpage-alfresco) [inline src/main/java/org/alfresco/rest/model/RestRuleModelsCollection.java:5]: I don't understand what this class is. It doesn't seem to be autogenerated and it doesn't seem to be used anywhere.

File: src/main/java/org/alfresco/rest/model/RestRuleSetModelsCollection.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package org.alfresco.rest.model;
+
+import org.alfresco.rest.core.RestModels;
+
+public class RestRuleSetModelsCollection extends RestModels<RestRuleSetModel, RestRuleSetModelsCollection>
```
Comment (tpage-alfresco) [inline src/main/java/org/alfresco/rest/model/RestRuleSetModelsCollection.java:5]: Same comment - what is this for?

File: src/main/java/org/alfresco/rest/model/RestRuleModelsCollection.java (hunk 1 above)
Comment (KikoUK) [reply src/main/java/org/alfresco/rest/model/RestRuleModelsCollection.java:5]: It wraps entries in a list with pagination - it is somehow generic way of wrapping entries

Some of other paging objects from yaml are manually created inside TAS project, see RestActionDefinitionModelsCollection and the class it extends

**Review by tpage-alfresco — APPROVED** [review]
LGTM

**Pull request merged**